src/content/developers/docs/apis/backend/index.md

```diff
@@ -42,6 +42,11 @@ These libraries abstract away much of the complexity of interacting directly wit
 
 - [cloudflare-eth.com](https://cloudflare-eth.com)
 
+**Coinbase Cloud Node -** **_Blockchain Infrastructure API._**
+
+- [Node](https://www.coinbase.com/cloud/products/node)
```

Comment:
```suggestion
- [Coinbase Cloud Node](https://www.coinbase.com/cloud/products/node)
```

Just going to change the name to make it more obvious like the others. Looks good otherwise!